portfolio/src/main/webapp/script.js

```diff
@@ -50,3 +44,29 @@ async function getMsgUsingAsyncAwait() {
   const msg = await response.text();
   document.getElementById('msg-container').innerText = msg;
 }
+*/
```

Comment:
I'd prefer we just delete these functions. If you ever need to get them back, you can look through the file history. That's part of the reason for version control.